rs/consensus/src/idkg/test_utils.rs

```diff
@@ -97,7 +103,13 @@ fn fake_signature_request_args(key_id: IDkgMasterPublicKeyId) -> ThresholdArgume
             key_id,
             message: Arc::new(vec![1; 48]),
         }),
-        MasterPublicKeyId::VetKd(_) => panic!("not applicable to vetKD"),
+        MasterPublicKeyId::VetKd(key_id) => ThresholdArguments::VetKd(VetKdArguments {
```

Comment:
This should probably be lifted out of the `idkg` submodule, but we can do this in a separate PR.

This likely applies to the other changes of `test_utils` as well.